doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,30 @@
+# QoS policies
+
+Unlike other middlewares like for example [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html) iceoryx tries to keep things simple and offers only basic Quality of Service (QoS) settings. The QoS settings are called options and can be used to optimize and tailor the communication.
```

Comment:
I do not like the wording "Unlike <others> iceoryx tries ... ". This is is too much subjective opinion and we should avoid it for this reason. 

It is more like state what we do and what it can be compared to, whether it is good or bad, complex or simple is for others to judge in my opinion. Furthermore I would not write "tries to", either we do it or we don't. There is no try ...

In short: this should be more matter of fact (less opinion) and no try.

Reply:
I personally would prefer this subjective comparison here as DDS has and will influence iceoryx heavily. Removed the try.